tensorflow/lite/micro/kernels/xtensa/xtensa_fully_connected.h

```diff
@@ -56,6 +57,84 @@ TfLiteStatus FullyConnectedEvalVision(TfLiteContext* context, TfLiteNode* node,
                                       TfLiteEvalTensor* output);
 #endif  // VISION_P6
 
+void* XtensaInitFullyConnected(TfLiteContext* context, const char* buffer,
+                               size_t length);
+
+TfLiteStatus XtensaEvalFullyConnectedQuantizedInt8(
+    TfLiteContext* context, TfLiteNode* node, const OpDataFullyConnected& data,
+    const TfLiteEvalTensor* input, const TfLiteEvalTensor* filter,
+    const TfLiteEvalTensor* bias, TfLiteEvalTensor* output);
+
+TfLiteStatus XtensaCalculateOpDataFullyConnected(
+    TfLiteContext* context, TfLiteFusedActivation activation,
+    TfLiteType data_type, const TfLiteTensor* input, const TfLiteTensor* filter,
+    const TfLiteTensor* bias, TfLiteTensor* output, OpDataFullyConnected* data);
+
+template <TfLiteType T = kTfLiteVariant>
+TfLiteStatus XtensaPrepareFullyConnected(TfLiteContext* context,
+                                         TfLiteNode* node) {
+  TFLITE_DCHECK(node->user_data != nullptr);
```

Comment:
once function no longer needs to be a template, we can move into xtensa/fully_connected_common.cc?